lemarche/templates/frontend/home/home.html

```diff
@@ -1,6 +1,85 @@
 {% extends "frontend/base.html" %}
 {% load static %}
 
-{% block title %}Les emplois de l'inclusion{% endblock %}
+{% block title %}API marché de l'inclusion{% endblock %}
 
 {% block content %}
+    <nav class="navbar navbar-expand-lg bg-warning text-dark">
+        Service en expérimentation et en construction
+    </nav>
+    <div class="container">
+       <div class="row no-gutters pt-3">
+           <div class="col col-lg-10 offset-lg-1 mt-4">
+            <div class="jumbotron">
+                <h1>API marché de l'inclusion</h1>
+                <p class="lead">L'API du <a href="https://lemarche.inclusion.beta.gouv.fr" target="_blank">marché de l'inclusion</a> met à disposition le répertoire qualifié des structures d'insertion par l'activité économique.</p>
+                <p>Cet API est encore en construction, et est appellée à évoluer.</p>
+                <p>
+                <p class="text-muted">Version: 0.5</p>
+            </div>
+            <div class="row mt-5">
+                <div class="col-12 bg-info text-white p-3">
+                    L'accès aux données complètes est limité pendant la période de construction du service.<br />
+                    Si vous désirez obtenir un token utilisateur, merci de nous addresser votre demande sur
```

Comment:
```suggestion
                    Si vous désirez obtenir un jeton utilisateur, merci de nous adresser votre demande sur
```

:pray: :fr: 
On va encore dire que je suis à cheval sur les mots mais j'assume.